src/main/java/com/user_service/DTOs/ConfirmForgotPasswordResponseDto.java

```diff
@@ -0,0 +1,12 @@
+package com.user_service.DTOs;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import software.amazon.awssdk.services.cognitoidentityprovider.model.ConfirmForgotPasswordResponse;
+
+@Data
+@AllArgsConstructor
+public class ConfirmForgotPasswordResponseDto {
+    ConfirmForgotPasswordResponse confirmForgotPasswordResponse;
```

Comment:
But we should be careful what we put there. Ideally, those should be POJO or our domain models. Right now, we're coupling heavily to Cognito. This should rather be avoided. If in the future we changed Cognito to sth else. we would need to change the logic of all other dependant classes.